src/System.Management.Automation/engine/remoting/fanin/WSManTransportManager.cs

```diff
@@ -1034,7 +1034,7 @@ internal override void StartReceivingData()
         /// <exception cref="PSRemotingTransportException">
         /// WSManCreateShellEx failed.
         /// </exception>
-        internal override void CreateAsync()
+        public override void CreateAsync()
         {
             Dbg.Assert(!isClosed, "object already disposed");
             Dbg.Assert(!string.IsNullOrEmpty(ConnectionInfo.ShellUri), "shell uri cannot be null or empty.");
```

Comment:
Since the `WSManClientSessionTransportManager` class itself is internal access, I will leave the asserts as they are.  My main concern is introducing a regression in this old code by changing it, although it seems unlikely.